tests/ui-components/Button/__snapshots__/Button.spec.tsx.snap

```diff
@@ -0,0 +1,76 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Button should render button with correct styles 1`] = `
+<View
+  accessibilityRole="button"
+  accessibilityState={
+    {
+      "busy": undefined,
+      "checked": undefined,
+      "disabled": undefined,
+      "expanded": undefined,
+      "selected": undefined,
+    }
+  }
+  accessibilityValue={
+    {
+      "max": undefined,
+      "min": undefined,
+      "now": undefined,
+      "text": undefined,
+    }
+  }
+  accessible={true}
+  collapsable={false}
+  focusable={true}
+  onClick={[Function]}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+  style={
+    {
+      "opacity": 1,
+    }
+  }
+>
+  <View
+    style={
+      [
+        {},
+        [
+          {
+            "backgroundColor": "#F0F2F3",
+            "borderRadius": 8,
+          },
+          [
+            {},
+          ],
+        ],
+      ]
+    }
+  >
+    <Text
+      style={
+        [
+          {
+            "borderColor": "#5A31F4",
+            "borderRadius": 8,
+            "borderStyle": "solid",
+            "borderWidth": 1,
+            "color": "#5A31F4",
+            "fontSize": 18,
+            "fontWeight": "bold",
+            "padding": 8,
+            "textAlign": "center",
+          },
+        ]
+      }
+    >
+      Login
+    </Text>
```

Comment:
Potrzebne są te style w testach?

Reply:
To sa snapshoty, gwnerowane sa automatycznie przy odpaleniu testu, jest to zrzut jak wyglada DOM w czasie testu, kiedy cos zmienisz w komponencie np w stylach to test ci jebnie i wtedy musisz zobaczyc czemu to jebnelo i mozesz zaznaczyc zeby wygenerowac nowy snapshot. To sa componenty typowo UI'owe wiec takie testowanie twgo jest imo najlepsze